docker-app/qfieldcloud/core/staticfiles/css/sso.css

```
@@ -6,6 +6,8 @@
   }
 
   .provider-button {
+    background-color: white;
```

```suggestion
    color: grey;
    background-color: white;
```

I guess we want to assure readable text over the white background too?